_projects/2024/100431421/100431421.Rmd

```diff
@@ -0,0 +1,666 @@
+    size = 1.5
+  )+
+  coord_flip() +  # Horizontal 
+  labs(
+    title = "Countries with less immigrants",
+  ) +
+  scale_y_continuous(expand = expansion(mult = c(0.05, 0.05)))+ #Space between bars
+  theme_minimal() +
+  theme(
+    axis.text.x = element_blank(),  # Eliminate text in axis X
+    axis.text.y = element_text(size = 5,  margin = margin(r = 5), family = "raleway"),   
+    axis.title.x = element_blank(),  # Eliminate the title in axis X
+    axis.title.y = element_blank(),  # Eliminate the title in axis X
+    panel.grid = element_blank(),
+    axis.line = element_blank(),
+    legend.position = "none",
+    aspect.ratio = 1.8,# More or less compressed
+    plot.title = element_text(
+      size = 4.5,        
+      face = "bold",
+      family = "raleway",
+      hjust = 0.5       
+    )
+  )
+
+# With the flags
+graph_2 <- bar_plot2 +
+  annotation_raster(flag_china, xmin = 4.7, xmax = 5.35, ymin = -0.9, ymax = -0.2)+
+  annotation_raster(flag_vietnam, xmin = 3.7, xmax = 4.35, ymin = -0.9, ymax = -0.2)+
+  annotation_raster(flag_cuba, xmin = 2.7, xmax = 3.35, ymin = -0.9, ymax = -0.2) +
+  annotation_raster(flag_madagascar, xmin = 1.7, xmax = 2.35, ymin = -0.9, ymax = -0.2) +
+  annotation_raster(flag_indonesia, xmin = 0.7, xmax = 1.35, ymin = -0.9, ymax = -0.2) +
+  coord_flip(clip = "off")
+
+graph_2
+```
+
+**IMPORTANT**: Just as before the barplots look a bit "ugly" because the flags appear on top of the text blocking it and also because the letters and numbers look tiny, but in the final plot the bar plots appear adjusted
+
+
+# FINAL PLOT WITH IMPROVEMENTS (BEFORE IÑAKI'S ADVICE)
+
+Now we want to join the map with both barplots. We do the same as before:
+
+```{r}
+# We load the images and create grobs
+map_grob <- ggplotGrob(map_2)
+graph_grob <- ggplotGrob(graph_1)
+graph2_grob <- ggplotGrob(graph_2)
+
+# We bind map + barplot1 + barplot2 + image
+final_plot <- ggdraw() +
+  draw_grob(map_grob) +  
+  draw_grob(graph_grob, x = 0.58, y = 0.05, width = 0.24, height = 0.24) +
+  draw_grob(graph2_grob, x = 0.4, y = 0.05, width = 0.24, height = 0.24) +
+  draw_grob(
+    textGrob(
+      "Source:", 
+      x = 0.1, y = 0.125, 
+      gp = gpar(fontface = "bold", fontsize = 6, fontfamily = "raleway"),
+      hjust = 0
+    )
+  ) +
+  draw_grob(
+    textGrob(
+      "Pew Research Center (2017) / World Bank (2017) /", 
+      x = 0.1, y = 0.1, 
+      gp = gpar(fontsize = 6, fontfamily = "raleway"),
+      hjust = 0
+    )
+  )+
+  draw_grob(
+    textGrob(
+      "Migration Report (2017) / GbMx (2017)", 
+      x = 0.1, y = 0.075, 
+      gp = gpar(fontsize = 6, fontfamily = "raleway"),
+      hjust = 0
+    )
+  ) 
+# We add the title
+plot <- plot_grid(
+  ggdraw() + 
+    draw_text("Global Map of Migration", size = 26, x = 0.08, y = 0.3, hjust = 0, fontface = "bold") +
+    draw_text("% of immigrants and flows of migration (2017)", size = 18, x = 0.08, y = 0.01, hjust = 0, fontface = "bold", color = "#444444"),
```

Comment:
This line goes out of view, insert a new line somewhere in the middle to avoid this.